src/constants/geospatial.constant.ts

```diff
@@ -0,0 +1,16 @@
+export const GEOSPATIAL = {
+  DEFAULT: {
+    RESULT_LANGUAGE: 'EN',
+    DATASET: 'DPA',
+  },
+  EXAMPLES: {
+    POSTCODE: 'SW1A 2AQ',
+    ORGANISATION_NAME: 'CHURCHILL MUSEUM & CABINET WAR ROOMS',
+    ADDRESS_LINE_1: 'CLIVE STEPS KING CHARLES STREET',
+    LOCALITY: 'LONDON',
+  },
+  REGEX: {
+    // UK postcode regex is from DTFS project and slightly optimised by lint.
```

Comment:
Please add link to the DTFS RegEx here for future reference.

Reply:
Added link to DTFS.
FYI DTFS has regex in 3 files, with 2 slightly different versions.